### PR TITLE
Gl qcom image processing

### PR DESCRIPTION
**Pull request by wooyoungqcom**
[extension]  GL_QCOM_image_processing support

### CODE REVIEW

File: glslang/MachineIndependent/Initialize.cpp [hunk 1]
```diff
@@ -10030,7 +10058,7 @@ void TBuiltIns::identifyBuiltIns(int version, EProfile profile, const SpvVersion
         if (profile != EEsProfile && version >= 460) {
             symbolTable.relateToOperator("reportIntersectionNV", EOpReportIntersection);
             symbolTable.relateToOperator("reportIntersectionEXT", EOpReportIntersection);
-	}
+	      }
```
Comment (arcady-lunarg) [inline glslang/MachineIndependent/Initialize.cpp:10061]: Stray whitespace diff

File: SPIRV/GlslangToSpv.cpp [hunk 2]
```diff
@@ -9568,6 +9605,20 @@ void TGlslangToSpvTraverser::addMeshNVDecoration(spv::Id id, int member, const g
     }
 }
 
+void TGlslangToSpvTraverser::addImageProcessingQCOMDecoration(spv::Id id, spv::Decoration decor)
+{
+  spv::Id opc = builder.getOpCode(id);
```
Comment (arcady-lunarg) [inline SPIRV/GlslangToSpv.cpp:9610]: This type should be spv::Op

**Review by arcady-lunarg — CHANGES_REQUESTED** [review]
Looks fine overall, just a couple of minor nits that need to be addressed before this is merged.

File: SPIRV/GlslangToSpv.cpp (hunk 2 above)
Comment (wooyoungqcom) [reply SPIRV/GlslangToSpv.cpp]: Thanks, much. I updated the branch with the fixes.

**Review by arcady-lunarg — APPROVED** [review]
LGTM, thanks.

**Pull request merged**